vesc_hw_interface/src/vesc_servo_controller.cpp

```diff
@@ -209,14 +222,14 @@ bool VescServoController::calibrate(const double position_current)
     return false;
   }
 
-  step++;
+  calibration_steps_++;
 
-  if (step % 20 == 0)
+  if (calibration_steps_ % 20 == 0)
   {
-    if (position_current == position_previous)
+    if (position_current == calibration_previous_position_)
     {
       // finishes calibrating
-      step = 0;
+      calibration_steps_ = 0;
       zero_position_ = position_current - calibration_position_;
       position_sens_ = calibration_position_;
       position_sens_previous_ = calibration_position_;
```

Comment:
Why is it different from this?
https://github.com/sbgisen/vesc/blob/dc6b62055cef148e0524d16b4da184705841972e/vesc_hw_interface/src/vesc_servo_controller.cpp#L90

Reply:
That's a very good question. I probably missed that line on #48.